### ISSUE TITLE
AWS Policy creation

### ISSUE THREAD

**Issue by gesch** [docker/labs#238]
Which lab is it that you're having issues with?
Docker for Java Developers

In Step 3 of the preperation:
Amazon Web Services credentials with following permissions. This is only needed for some parts of the workshop.

The AWS setup failed for some reason because of the maximum policy length. The happens when creating a custom policy manually in AWS. 

Error message:
An error occurred: Cannot exceed quota for PolicySize: 5120

**Comment by ManoMarks** [comment]
We need a lot more info. Please follow the new issue template. Not even sure which lab this is:

** PLEASE ONLY USE THIS ISSUE TRACKER TO SUBMIT ISSUES WITH THE DOCKER LABS TUTORIAL CONTENT **

* If you have a bug working with Docker itself, not related to these labs, please file the bug on the [Docker repo](https://github.com/docker/docker) *
* If you would like general support figuring out how to do something with Docker, please use the Docker Slack channel. If you're not on that channel, sign up for the [Docker Community](http://dockr.ly/MeetUp) and you'll get an invite. *
* Or go to the [Docker Forums](https://forums.docker.com/) *

Please provide the following information so we can assess the issue you're having

**Which lab is it that you're having issues with?**


**Description**



**Steps to reproduce the issue, if relevant:**
1.
2.
3.

**Describe the results you received:**


**Describe the results you expected:**


**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

```
(paste your output here)
```

**Output of `docker info`:**

```
(paste your output here)
```

**Additional environment details (AWS, Docker for Mac, Docker for Windows, VirtualBox, physical, etc.):**

**Comment by arush-sal** [comment]
@gesch The error mentioned is generally caused due to an excessively lengthy group policy length. As described [here](http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-limits.html) you cannot have a group policy that is longer than 5120 characters. It is an AWS limitation rather than Docker issue.

**Comment by arush-sal** [comment]
@ManoMarks This can be closed as this is an AWS related error rather than a Docker lab error.

**Comment by gesch** [comment]
@arush-sal you are right, it can be closed, but you should fix the documentation because the provided policy can not be imported because of the AWS limitation.

**Comment by arush-sal** [comment]
@gesch Maybe I'll be able to fix that if you can specify the issue as requested above.

**Comment by gesch** [comment]
@arush-sal @arun-gupta @ManoMarks sorry, it takes way more time explaining the problem for an invalid documentation than ignoring it. You will get the same issue in the next Java workshop using the lessons from the dockerCon 2017. Sorry for wasting your and my time. Tried to be helpful getting this ironed out as requested in the running session by Arun. Not sure what you need more. The workshop track and the lesson file is in my option a solid information. And to be honest, there is only one AWS policies information in all the documents provided to the attendees for the workshop.

**Comment by arun-gupta** [comment]
@gesch +1, attendees will encounter this error. And this is is on my list to find a solution to.

@ManoMarks can you please reopen the bug?
